frameworks/hdfs/src/main/dist/krb5.conf

```diff
@@ -6,7 +6,7 @@ udp_preference_limit = 1
 
 [realms]
   LOCAL = {
```

Comment:
The templating of this is addressed in #1893. Ensuring the correct template is used here.